lib/bloc/account/account_bloc.dart

```diff
@@ -231,6 +238,68 @@ class AccountBloc {
     action.resolve(await _breezLib.sendWalletCoins(action.destAddress, Int64(action.feeRate)));     
   }
 
+  Future _exportPaymentsAction(ExportPayments action) async {
```

Comment:
Also I think we can use a helper class `CsvExporter` to implement this action in order not to add too much low level code to the AccountBloc. The use can be something like this (or similar):
`new CsvExporter(paymentItems, filter).export()`

Reply:
[Done](https://github.com/breez/breezmobile/pull/203/commits/8e33d0e1551941c358957944da9f283ca8287816)